src/components/Pad/Pad.js

```diff
@@ -4,36 +4,45 @@ import styled from 'styled-components/macro'
 
 Pad.propTypes = {
   name: PropTypes.string.isRequired,
+  index: PropTypes.number.isRequired,
+  onTouchStart: PropTypes.func.isRequired,
+  onTouchEnd: PropTypes.func.isRequired,
   player: PropTypes.object,
```

Comment:
Player ist later not requiered when users can load their own sounds. Maybee the user want to use only 8 instead of 16 sounds, so only 8 players are needed.